benchmark/solver/solver.cpp

```diff
@@ -59,14 +59,20 @@ DEFINE_uint32(max_iters, 1000,
 
 DEFINE_double(rel_res_goal, 1e-6, "The relative residual goal of the solver");
 
-DEFINE_string(solvers, "cg",
-              "A comma-separated list of solvers to run. "
-              "Supported values are: bicgstab, cg, cgs, fcg, gmres, overhead");
+DEFINE_string(
+    solvers, "cg",
```

Comment:
We definitely need to define a clang-format standard here. Otherwise, we keep switching between formatting

Reply:
I think this formatting change is correct, since the last line would otherwise exceed 80 characters